src/planscape/collaboration/views.py

```diff
@@ -46,3 +51,21 @@ def post(self, request, format=None):
             logger.exception("Something failed during the creation of a new invite.")
             # this will return a 500
             raise
+
+
+class GetInvitationsForObject(APIView):
+    def get(self, request: Request, target_entity: str, object_pk: int):
+        user = request.user
+        content_type = ContentType.objects.get(model=target_entity)
+        Model = content_type.model_class()
+        instance = Model.objects.get(pk=object_pk)
+
+        if not CollaboratorPermission.can_add(user, instance):
```

Comment:
ahg sorry, i messed up this is probably can_view, no? This method gets the list of invites, not adding new invites

Reply:
yes, but we'll check for that on the endpoint that adds, no?

Reply:
> BUT, he needs to be able to add to see the modal no?
> yes, but we'll check for that on the endpoint that adds, no?

I mean technically no. Just happens that on our role structure there are no roles that only can see the modal, but those actions (viewing the list of invites and adding new invites) can be considered separate things